src/community/design-system-working-group/index.md.njk

```diff
@@ -20,8 +20,14 @@ By including representatives from a mixture of disciplines and departments, the
 
 It means that decisions made are fair and unbiased, and that the Design System reflects the experiences of the whole of government not just one department. 
 
-## How to join the working group
+## Join the working group
 
-The GOV.UK Design System team is looking into the best way of helping people to join the working group.
+We’re looking for new members to help make the working group more diverse and better represent the wider public sector.
 
-In the meantime if you are interested in joining the working group email the Design System team at govuk-design-system-support@digital.cabinet-office.gov.uk
+If you think you can expand our range of voices across ethnicity, age, gender, experience and discipline, we want to hear from you.
+
+You’ll help promote collaboration across government and the Digital Transformation Strategy. Although our members volunteer their time with us, many of them do so as part of their community or corporate objectives.
+
+Find out more about what’s involved by reading our [Working group principles & ways of working](https://docs.google.com/document/d/1f6jUvxs94LjZXvHxiJIitm6_daKy2eYaVNVlYxmz5OE/edit?usp=sharing) document.
+
+If you’re interested, [sign up and tell us a bit about yourself](#). The deadline is 5pm on 26 February 2021.
```

Comment:
Will this link to the 'get in touch' page or something else?

Reply:
Yes, Imran's working on a sign-up form which will be linked from here.

Ideal user journey would be something like:
- User sees post on social media or email
- Clicks on link to this landing page
- Finds information they need about the working group
- Clicks through to sign-up form and applies